### ISSUE TITLE
"show in qualifier" shows when not really in qualifier

### ISSUE THREAD

**Issue by ezio416** [MisfitMaid/tm-map-info#27]
Feel free to close if there's no good solution. I notice this in at least 2 other plugins (including one of mine) where if you join the TOTD server shortly after qualifier ends, the gamemode is still `TM_COTDQualifications_Online` even though it's a lie. Maybe the time could be checked since qualifier only lasts 15 minutes but the server seems to stay in that mode for about 30.

![Screenshot 2024-03-16 202727](https://github.com/MisfitMaid/tm-map-info/assets/37951898/c9c7d025-b109-4719-b9ed-14a288a98572)

**Comment by XertroV** [comment]
oh interesting. yeah odd edge case.

**Comment by ezio416** [comment]
This can be resolved with MLFeed - only displaying when the gamemode matches and `MLFeed::GetRaceData_V4().COTDQ_QualificationsProgress == MLFeed::QualificationStage::Running` will do it. However I'm not sure adding the dependency is worth it just for this

**Comment by XertroV** [comment]
another way might be to look for both UI elements and decide based on which is visible

**Comment by XertroV** [comment]
Hopefully I have a fix in 68fdce3
(testing now)

**Comment by XertroV** [comment]
yep confirmed https://github.com/XertroV/tm-map-info/commit/68fdce3def2dcacb5c1a75b439442656c01bfe2d fixes it
